src/packages/eslint-plugin-eigenspace-redux/package.json

```diff
@@ -0,0 +1,13 @@
+{
+    "name": "eslint-plugin-eigenspace-redux",
+    "version": "0.0.14",
+    "description": "Eigenspace team's codestyle plugin for redux",
+    "license": "MIT",
+    "dependencies": {
+        "eslint-plugin-react-redux": "3.0.1"
+    },
+    "peerDependencies": {
+        "react": "16.x",
```

Comment:
It would be nice to check whether all the rules have such a restriction on the version of the reaction